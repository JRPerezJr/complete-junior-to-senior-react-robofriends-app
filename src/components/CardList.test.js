import React from 'react';
import { shallow } from 'enzyme';
import CardList from './CarList';

describe('<CardList/> component tests', () => {
  it('sanity test', () => {
    expect(true).toBe(true);
  });

  it('renders CardList component', () => {
    const mockRobots = [
      {
        id: 1,
        name: 'James',
        username: 'jbond',
        email: 'jbond@universalexports.com',
      },
    ];
    expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
  });
});
