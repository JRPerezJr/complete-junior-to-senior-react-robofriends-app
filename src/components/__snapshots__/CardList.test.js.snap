// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardList/> component tests renders CardList component 1`] = `ShallowWrapper {}`;
